typings/internal-electron.d.ts

```diff
@@ -138,12 +138,16 @@ declare namespace Electron {
     acceleratorWorksWhenHidden?: boolean;
   }
 
-  interface IpcMainEvent {
+  interface ReplySender {
```

Comment:
This hurts me, I can't explain why, or what I'd change it to, I just want it on the record 😆